autogpt_platform/backend/check_logging.py

```diff
@@ -0,0 +1,17 @@
+import pathlib
+import re
+
+PATTERN = re.compile(r"logging\.getLogger\(")
+ROOT = pathlib.Path(__file__).resolve().parent / "backend"
+
+errors = []
+
+for path in ROOT.rglob("*.py"):
+    if path.name == "logging.py":
+        continue
+    text = path.read_text()
+    if PATTERN.search(text) and "TruncatedLogger" not in text:
+        errors.append(str(path.relative_to(ROOT)))
+
+if errors:
+    print("Plain logging.getLogger usage detected in:\n" + "\n".join(errors))
```

Comment:
So any usage of `logger.getLogger` not wrapped in `TruncatedLogger` is now reported as a linting error?

But this PR doesn't fix all usages of `logging.getLogger` so the PR wouldn't pass this check. 

*If* we needed system-wide logging truncation, add some middleware in `configure_logging` to do this rather than wrapping all uses across the codebase in a helper class. 

But the last time we discussed this, there was no real need for this, because there are only a few specific places where messages with long "attachments" are being logged.